PowerShellTools/DebugEngine/ScriptProperty.cs

```diff
@@ -114,16 +110,19 @@ public class ScriptProperty : IDebugProperty2
 
         protected readonly ScriptDebugger _debugger;
 
-        public ScriptProperty(ScriptDebugger debugger, string name, string value, string type)
+        protected string _path { get; set; }

```

Comment:
Can path ever change after initialization?  Why not protected Path{ get; private set;} with it being set inside the base(ScriptDebugger debugger, Variable var, NewPathStuffHere) constructor?